packages/browser-wallet/src/popup/shared/utils/account-helpers.ts

```diff
@@ -17,7 +17,7 @@ export function useIdentityOf(cred?: WalletCredential) {
         if (!cred) {
             return undefined;
         }
-        return identities.find(isIdentityOfCredential(id));
+        return identities.find((id) => isIdentityOfCredential(id)(cred));
```

Comment:
I was looking very hard at this previously, as I did not understand how `isIdentityOfCredential` would work here. Hehe...

Reply:
Yeah :upside_down_face: